assigner/commands/score.py

```diff
@@ -409,7 +410,7 @@ def setup_parser(parser: argparse.ArgumentParser):
             "--files",
             nargs="+",
             dest="files",
-            default=["grade.sh", ".gitlab-ci.yml"],
+            default=[".gitlab-ci.yml"],
```

Comment:
Do you think enough people are using `assigner score` right now that I should make a point of describing this change and how to get previous behavior in the announcement email, or is a quick note fine?

Reply:
I don't think so - @BehindTheBrain would be able to confirm but I don't think `grade-sh` has a significant number of users.